### PR TITLE
Fixed windows buttons for Firefox 117

### PR DESCRIPTION
**Pull request by easonwong-de**
Sorry for the delay.

### CODE REVIEW

**Comment by kayoisdumb** [comment]
Tysm. This has been pissing me off.

File: chrome/WhiteSur/parts/csd.css [hunk 1]
```diff
@@ -54,6 +54,20 @@ GNOME CSD styles for headerbar on Firefox [tabsintitlebar] @namespace xul url("h
 	top:10px !important;
 }
 
+/* Fix the botton order */
```
Comment (AdamXweb) [inline chrome/WhiteSur/parts/csd.css:57]: Could this be moved to the /parts/windows.css specific CSS not to conflict with other users?

**Review by AdamXweb — CHANGES_REQUESTED** [review]
Looks good! I will need to follow up with the custom modification 'windows-swapclose.css' to adjust the order. Could the fix be moved to the windows.css file and i'll merge!

File: chrome/WhiteSur/parts/csd.css (hunk 1 above)
Comment (easonwong-de) [reply chrome/WhiteSur/parts/csd.css:57]: As I look into it, many of the CSS that I added are already existent in `windows.css`. I think maybe `@media screen and (-moz-windows-compositor)` doesn’t work any more. Deleting the media tag might fix the problem and this pull request is no longer needed.

**Review by AdamXweb — APPROVED** [review]

**Comment by AdamXweb** [comment]
Updated files to continue support for MacOS and Windows

File: chrome/WhiteSur/parts/csd.css (hunk 1 above)
Comment (AdamXweb) [reply chrome/WhiteSur/parts/csd.css]: Thanks for picking that up. I've updated from the media query to the new query targeting windows via `@media  (-moz-platform: windows)` instead. Merging with master

**Pull request merged**